static/src/javascripts/projects/commercial/modules/header-bidding/slot-config.spec.ts

```diff
@@ -48,27 +43,31 @@ const slotPrototype = {
 	defineSizeMapping: () => slotPrototype,
 	addService: () => slotPrototype,
 	setTargeting: () => slotPrototype,
-    setSafeFrameConfig: () => slotPrototype
+	setSafeFrameConfig: () => slotPrototype,
 };
 
 // Mock window.googletag
 window.googletag = {
 	sizeMapping: () => ({
-        addSize: () => {},
+		// @ts-expect-error these are just mocks
+		addSize: () => {
+			/* Do nothing*/
+		},
 		build: () => [],
 	}),
+	// @ts-expect-error these are just mocks
 	defineSlot: () => ({ ...slotPrototype }),
+	// @ts-expect-error these are just mocks
 	pubads: () => ({}),
 };
```

Comment:
Decided to leave a few `@ts-expect-errors` in here - I don't think there's much value trying to type these mocks when they're ill-typed from the get go. Let me know if you think there's a better solution